scripts/ortoa-lib.sh

```diff
@@ -52,21 +52,6 @@ _EOF_
 ############################################
 
 ortoa-client-run() {
-    local HELP="""\
-Run the ORTOA client
-
-Syntax: ortoa-client-run [-h]
-----------------------------------------------
-    -h                  Print this help message
-"""
-    OPTIND=1
-    while getopts ":h" option; do
-        case "${option}" in
-            h) echo "${HELP}"; return 0 ;;
-        esac
-    done
```

Comment:
I would leave the local HELP = ... but remove lines 62-67. This is assuming that -h will cause a real help text to be printed to stdout

Reply:
Sure, re-added client help text